bin/pfurl

```diff
@@ -160,8 +160,15 @@ parser.add_argument(
     default= '',
     help   = 'any web address'
 )
+parser.add_argument(
+    '--insecure',
+    action = 'store_true',
+    dest   = 'insecure',
+    default= False,
+    help   = 'Indicates that the this message is from a trusted source, enabling https servers to accept self signed certificates'
```

Comment:
Bad sentence

Reply:
suggested edit: 'allow insecure connections and self signed certificates over https'